storage/pkg/flagset/storageocdata.go

```diff
@@ -1,111 +0,0 @@
-package flagset
-
-import (
-	"github.com/micro/cli/v2"
-	"github.com/owncloud/ocis/storage/pkg/config"
-)
-
-// StorageOCDataWithConfig applies cfg to the root flagset
-func StorageOCDataWithConfig(cfg *config.Config) []cli.Flag {
-	flags := []cli.Flag{
-
-		// debug ports are the odd ports
-		&cli.StringFlag{
-			Name:        "debug-addr",
-			Value:       "0.0.0.0:9165",
-			Usage:       "Address to bind debug server",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_DATA_DEBUG_ADDR"},
-			Destination: &cfg.Reva.StorageOCData.DebugAddr,
-		},
-
-		// Services
-
-		// Storage oc data
-
-		&cli.StringFlag{
-			Name:        "network",
-			Value:       "tcp",
-			Usage:       "Network to use for the storage service, can be 'tcp', 'udp' or 'unix'",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_DATA_NETWORK"},
-			Destination: &cfg.Reva.StorageOCData.Network,
-		},
-		&cli.StringFlag{
-			Name:        "protocol",
-			Value:       "http",
-			Usage:       "protocol for storage service, can be 'http' or 'grpc'",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_DATA_PROTOCOL"},
-			Destination: &cfg.Reva.StorageOCData.Protocol,
-		},
-		&cli.StringFlag{
-			Name:        "addr",
-			Value:       "0.0.0.0:9164",
-			Usage:       "Address to bind storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_DATA_ADDR"},
-			Destination: &cfg.Reva.StorageOCData.Addr,
-		},
-		&cli.StringFlag{
-			Name:        "url",
-			Value:       "localhost:9164",
-			Usage:       "URL to use for the storage service",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_DATA_URL"},
-			Destination: &cfg.Reva.StorageOCData.URL,
-		},
-		&cli.StringSliceFlag{
-			Name:    "service",
-			Value:   cli.NewStringSlice("dataprovider"),
-			Usage:   "--service dataprovider [--service otherservice]",
-			EnvVars: []string{"STORAGE_STORAGE_OC_DATA_SERVICES"},
-		},
-		&cli.StringFlag{
-			Name:        "driver",
-			Value:       "owncloud",
-			Usage:       "storage driver for oc data mount: eg. local, eos, owncloud, ocis or s3",
-			EnvVars:     []string{"STORAGE_STORAGE_OC_DATA_DRIVER"},
```

Comment:
used in drone https://github.com/owncloud/ocis/blob/d422a91a01b98893d0a10a7555092342755ee33b/.drone.star#L549
not needed anymore?

Reply:
Yes, the storages got streamlined so there is no OC storage anymore.